src/libraries/System.Security.Cryptography.ProtectedData/src/System/Security/Cryptography/ProtectedData.cs

```diff
@@ -16,14 +16,20 @@ public static partial class ProtectedData
 
         public static byte[] Protect(byte[] userData, byte[]? optionalEntropy, DataProtectionScope scope)
```

Comment:
It's missing SupportedOSPlatformAttribute annotation

Reply:
Isn't this sufficient? https://github.com/vcsjones/runtime/blob/3d98834cd3cf6d0ce017c74cdd1dd32bacd0d5de/src/libraries/System.Security.Cryptography.ProtectedData/Directory.Build.props#L5

Reply:
When I look at the library in the locally built package, I see the `SupportedOSPlatformAttribute("windows")` on the whole assembly:

```
  .custom instance void [System.Runtime]System.Runtime.Versioning.SupportedOSPlatformAttribute::.ctor(string) = ( 01 00 07 77 69 6E 64 6F 77 73 00 00 )             // ...windows..
```

So think we're fine.

Reply:
missed that, so we should be fine.